app/client/views/confirmation/confirmation.html

```diff
@@ -207,118 +197,34 @@
           </div>
 
           <br>
-          <br>
+          <br>-->
 
           <div class="divided title">Travel</div>
 
           <div class="field">
             <p>
-              As a reminder, our reimbursement policy is as follows:
+              VTHacks tries to send out buses to schools that send a lot of attendees. If you would like transportation via bus please mark the box below
             </p>
-            <ul>
-              <li> Up to $100 from within Massachusetts </li>
-              <li> Up to $200 from anywhere else in North America </li>
-              <li> Up to $500 from outside North America </li>
-            </ul>
             <p>
-              We’ll also be sending buses free of charge to Princeton, NJ; New York, NY; New Haven, CT; and Providence, RI. Again, we only reimburse the minimum-cost travel path. Head back to the dashboard to read our full travel policy.
+              If we can, we will send a bus your way. We do our best to accomodate but there is no guarantee there will be a bus at your school. We will release
+              a list of schools with buses closer to the hackathon and hopefully there is one close to you. Always feel free to send us an email at <a href="mailto:transportation@vthacks.com">transportation@vthacks.com</a> with any questions.
             </p>
           </div>
 
           <div class="field">
-            <label> Will you be needing travel reimbursement? </label>
+            <label> Would you like a bus at your school? </label>
             <div class="ui checkbox">
               <input type="checkbox" ng-model="user.confirmation.needsReimbursement">
```

Comment:
Changed variable to something like "user.confirmation.needsTransportation" and update that in the User model.